salt/orchestrate/edx/build_ami.sls

```diff
@@ -215,9 +215,6 @@ compile_assets_for_edx_{{ PURPOSE }}:
     - name: cmd.run
     - arg:
         - /edx/bin/edxapp-update-assets
-    - kwarg:
-        use_vt: True
```

Comment:
What does this do?

Reply:
That's a flag that is supposed to allow for streaming output back to the master rather than waiting until the end to see output from `cmd.run`. I was testing it to see if it would alleviate the timeouts but it had no effect so there's no reason to keep it.